#request
int32 address
bool level
---
#response
bool success
